enclave_apps/key_xor_test_app/src/main.rs

```diff
@@ -61,11 +57,10 @@ fn run_server() -> ! {
 
 #[alloc_error_handler]
 fn out_of_memory(layout: ::core::alloc::Layout) -> ! {
-    panic!("error allocating memory: {:#?}", layout);
+    oak_restricted_kernel_sdk::alloc_error_handler(layout);
 }
```

Comment:
I assume something like the following didn't work?

```rust
#[alloc_error_handler]
oak_restricted_kernel_sdk::alloc_error_handler
```

Reply:
to my dismay it didn't :/